### PR TITLE
SAK-32343 

### PR DESCRIPTION
**Pull request by axxter99**
Orginezed import
Add default set
raw type
import org.apache.commons.lang.StringUtils; => import org.apache.commons.lang3.StringUtils; 
Kernel cover => Kernel api

### CODE REVIEW

**Comment by axxter99** [comment]
cc @buckett

File: admin-tools/src/java/org/sakaiproject/archive/tool/ArchiveAction.java [hunk 1]
```diff
@@ -69,6 +69,10 @@
 public class ArchiveAction
 	extends VelocityPortletPaneledAction
 {
+	/**
```
Comment (buckett) [inline admin-tools/src/java/org/sakaiproject/archive/tool/ArchiveAction.java]: Drop the empty comment?

File: admin-tools/src/java/org/sakaiproject/authz/tool/PermissionsHelperAction.java [hunk 2]
```diff
@@ -65,6 +65,11 @@
  */
 public class PermissionsHelperAction extends VelocityPortletPaneledAction
 {
+	/**
```
Comment (buckett) [inline admin-tools/src/java/org/sakaiproject/authz/tool/PermissionsHelperAction.java]: Drop the empty comment?

File: admin-tools/src/java/org/sakaiproject/authz/tool/RealmsAction.java [hunk 3]
```diff
@@ -63,6 +63,11 @@
 public class RealmsAction extends PagedResourceActionII
 {
 
+	/**
```
Comment (buckett) [inline admin-tools/src/java/org/sakaiproject/authz/tool/RealmsAction.java]: Drop the empty comment?

File: admin-tools/src/java/org/sakaiproject/user/tool/UsersAction.java [hunk 4]
```diff
@@ -106,6 +104,11 @@
 @Slf4j
 public class UsersAction extends PagedResourceActionII
 {
+	/**
```
Comment (buckett) [inline admin-tools/src/java/org/sakaiproject/user/tool/UsersAction.java]: Drop the empty comment?

File: admin-tools/src/java/org/sakaiproject/user/tool/UsersAction.java [hunk 5]
```diff
@@ -364,14 +389,14 @@ else if (mode.equals("mode_helper") && StringUtils.equals(status, "processImport
 	private String buildListContext(SessionState state, Context context)
 	{
 		// put the service in the context
-		context.put("service", UserDirectoryService.getInstance());
+		context.put("service", ComponentManager.get(UserDirectoryService.class));
```
Comment (axxter99) [inline admin-tools/src/java/org/sakaiproject/user/tool/UsersAction.java]: Sping?

ResourcesAction.java (L 928)
	private AliasService aliasService;
	private AuthzGroupService authzGroupService;

	public ResourcesAction() {
		aliasService = ComponentManager.get(AliasService.class);
		authzGroupService = ComponentManager.get(AuthzGroupService.class);
	}

Comment (buckett) [inline admin-tools/src/java/org/sakaiproject/user/tool/UsersAction.java]: Eg:
```
context.put("service", userDirectoryService);
```

File: admin-tools/src/java/org/sakaiproject/user/tool/UsersAction.java [hunk 6]
```diff
@@ -539,8 +564,8 @@ private String buildEditContext(SessionState state, Context context)
 	{
 		
 		// put the service in the context
-		context.put("service", UserDirectoryService.getInstance());
-
+		context.put("service", ComponentManager.get(UserDirectoryService.class));
```
Comment (axxter99) [inline admin-tools/src/java/org/sakaiproject/user/tool/UsersAction.java]: ditto ..

**Pull request merged**